playbooks/vagrant-webtemp.yml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
We should delete this file.
It served its purpose originally, but now we should be installing the beats into existing servers (like devstack or other microservices) instead of onto a standalone/test system.